src/bot.ts

```diff
@@ -2,13 +2,24 @@
  * Deepindigo
  *
  */
-const { MOVE, SWITCH } = require('leftovers-again/src/decisions');
-
+import ai from 'leftovers-again/src/ai';
+const AI = ai.AI;
+import decisions from 'leftovers-again/src/decisions';
+type Decision = decisions.Decision;
+const MOVE = decisions.MOVE;
+const SWITCH = decisions.SWITCH;
+import battle from 'leftovers-again/src/battle';
+type State = battle.State;
+import pokemon from 'leftovers-again/src/model/pokemon';
+type PokemonData = pokemon.PokemonData;
```

Comment:
This is the only ugly part IMO

Reply:
we can fix by going back to the single .d.ts file i believe